platforms/ios/src/TangramMap/platform_ios.mm

```diff
@@ -115,17 +115,36 @@ void initGLExtensions() {
     return data;
 }
 
+bool allowedFamily(NSString* familyName) {
+    const NSArray<NSString *> *allowedFamilyList = @[ @"Hebrew", @"Kohinoor", @"Gumurki", @"Thonburi", @"Tamil",
+                                                    @"Gurmukhi", @"Kailasa", @"Sangam", @"PingFang", @"Geeza",
+                                                    @"Mishafi", @"Farah", @"Hiragino" ];
+
+    for (NSString* allowedFamily in allowedFamilyList) {
```

Comment:
I think this makes sense since font family names can contain additional characters beyond just what's been listed above.

However, just for informational sake: You can simplify this if you want to check for content *equality*. `NSArray` has a method called `containsObject:` which you can pass your `familyName` value to. This subsequently calls `isEqual` on each string and does a content-based (not pointer-based) comparison on the strings.